src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -421,6 +421,11 @@ private function fixNewline(Tokens $tokens, $index, $indentation, $override = tr
             return;
         }
 
+        $nextMeaningfulTokenIndex = $tokens->getNextMeaningfulToken($index);
+        if ($tokens[$nextMeaningfulTokenIndex]->equals(')')) {
```

Comment:
Should this change be really applied only for master branch ?

Reply:
The bug exists on branch 2.12 therefore the fix should be merged on that branch.